publish/publish_workflow.wdl

```diff
@@ -237,7 +237,7 @@ task prepGithub {
             --header "Authorization: token $(cat ~{githubTokenFile})" \
             --head \
             https://api.github.com/rate_limit \
-        | grep -i -F 'X-OAuth-Scopes:' || true > scopesHeader.txt
+        | grep -i -F 'X-OAuth-Scopes:' > scopesHeader.txt
```

Comment:
TOL This is totally fine, though I believe the following would preserve the original intent without the brokenness
```suggestion
        | { grep -i -F 'X-OAuth-Scopes:' || true; } > scopesHeader.txt
```